### ISSUE TITLE
Simple and elegant UI, some technical questions

### ISSUE THREAD

**Issue by lucasjinreal** [olusola-olaoye/Howfa-messenger#2]
May I ask some questions about your work?

1. Does the code using Bloc design?
2. What's the backbone server using?

**Comment by kharithomas** [comment]
I am sure this was already answered but: 

1. Bloc design is not being used
2. The server is Firebase, if you'd like to run the code on your machine simply replace the proper Firebase files to connect it with your Firestore database

**Comment by lucasjinreal** [comment]
@kharithomas  Well, so sad, firebase not accessable in China.

**Comment by kharithomas** [comment]
@jinfagang Yes, hopefully Firebase will expand to China in the near future. In the meantime, check out this article on Back4App, it may prove useful for you.

https://blog.usejournal.com/flutter-bloc-back4app-basic-chat-app-489e18e98052
